### ISSUE TITLE
use LORA to finetune BERT-crf

### ISSUE THREAD

**Issue by zhanghanweii** [huggingface/peft#417]
If I use LORA to fine-tune bert-crf, do I need to rewrite BertForTokenClassification, because the cross-entropy loss function is used in class BertForTokenClassification but not the crf loss function. And whether task_type is TOKEN_CLS

Thanks!
